buildenv/jenkins/variables/defaults.yml

```diff
@@ -114,49 +114,39 @@ cmake:
       - *defaultTests
       - special.system
 #========================================#
+# JITServer
+#========================================#
+jitserver:
+  extra_configure_options: '--enable-jitserver'
+  test_flags: 'JITAAS'
+#========================================#
+# OpenSSL
+#========================================#
+openssl:
+  extra_getsource_options: '--openssl-version=1.1.1d'
+  extra_configure_options: '--with-openssl=fetched'
+#========================================#
 # Linux PPCLE 64bits Compressed Pointers
 #========================================#
 ppc64le_linux:
+  extends: ['openssl']
   boot_jdk:
     8: '/usr/lib/jvm/adoptojdk-java-80'
     11: '/usr/lib/jvm/adoptojdk-java-11'
     12: '/usr/lib/jvm/adoptojdk-java-11'
     13: '/usr/lib/jvm/adoptojdk-java-12'
     next: '/usr/lib/jvm/adoptojdk-java-13'
   release:
+    all: 'linux-ppc64le-server-release'
     8: 'linux-ppc64le-normal-server-release'
     11: 'linux-ppc64le-normal-server-release'
-    12: 'linux-ppc64le-server-release'
-    13: 'linux-ppc64le-server-release'
-    next: 'linux-ppc64le-server-release'
   freemarker: '/home/jenkins/freemarker.jar'
```

Comment:
`freemarker` and `openjdk_reference_repo` could also be inherited. They have the same values for all specs except win and osx